src/mock.rs

```diff
@@ -19,16 +19,31 @@
 use crate as substratee_registry;
 use frame_support::parameter_types;
 use frame_system as system;
-use sp_core::{sr25519, H256};
+use sp_core::H256;
 use sp_keyring::AccountKeyring;
 use sp_runtime::{
-    testing::Header,
-    traits::{BlakeTwo256, IdentityLookup, Verify},
+    generic,
+    traits::{BlakeTwo256, IdentifyAccount, IdentityLookup, Verify},
 };
 use substratee_registry::Config;
 
-type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
-type Block = frame_system::mocking::MockBlock<Test>;
+pub type Signature = sp_runtime::MultiSignature;
+pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
+pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
```

Comment:
Aligned the types here with the ones from Integritee parachain to use the same code for the tests as for the benchmarks.